Source/Flow/Private/Nodes/World/FlowNode_CallOwnerFunction.cpp

```diff
@@ -368,27 +387,20 @@ UClass* UFlowNode_CallOwnerFunction::GetParamsClassForFunctionName(const UClass&
 
 bool UFlowNode_CallOwnerFunction::TryAllocateParamsInstance()
 {
-	FObjectPropertyBase* ParamsProperty = FindFProperty<FObjectPropertyBase>(GetClass(), GET_MEMBER_NAME_CHECKED(UFlowNode_CallOwnerFunction, Params));
-	check(ParamsProperty);
-
-	UClass* RequiredParamsClass = GetRequiredParamsClass();
-
-	if (ParamsProperty)
-	{
-		// Update the property to filter for just this class (and its subclasses)
-		ParamsProperty->SetPropertyClass(RequiredParamsClass);
-	}
-
 	if (FunctionRef.GetFunctionName().IsNone())
 	{
+		// Throw out the old params object (if any)
+		Params = nullptr;
+
 		return false;
 	}
 
 	const UClass* ExistingParamsClass = GetExistingParamsClass();
+	UClass* RequiredParamsClass = GetRequiredParamsClass();
 
 	const bool bNeedsAllocateParams = 
 		!IsValid(ExistingParamsClass) ||
-		!ExistingParamsClass->IsChildOf(RequiredParamsClass);
+		ExistingParamsClass != RequiredParamsClass;
```

Comment:
Requiring explicit class match now.  This is required by the new approach for allocating the correct params object for a CallOwner function.